packages/manager/src/features/CloudPulse/Widget/components/CloudPulseAggregateFunction.tsx

```diff
@@ -52,26 +55,31 @@ export const CloudPulseAggregateFunction = React.memo(
     ] = React.useState<AggregateFunction>(defaultValue);
 
     return (
-      <StyledWidgetAutocomplete
-        isOptionEqualToValue={(option, value) => {
-          return option.label == value.label;
-        }}
-        onChange={(e, selectedAggregateFunc: AggregateFunction) => {
-          setSelectedAggregateFunction(selectedAggregateFunc);
-          onAggregateFuncChange(selectedAggregateFunc.label);
-        }}
-        textFieldProps={{
-          hideLabel: true,
-        }}
-        autoHighlight
-        disableClearable
-        fullWidth={false}
-        label="Select an Aggregate Function"
-        noMarginTop={true}
-        options={availableAggregateFunc}
-        sx={{ width: '100%' }}
-        value={selectedAggregateFunction}
-      />
+      <CloudPulseTooltip title={'Aggregation function'}>
+        <StyledWidgetAutocomplete
+          getOptionLabel={(option: { label: string }) => {
```

Comment:
```suggestion
          getOptionLabel={(option) => {
```

We shouldn't need to put a type here. The typescript should be able to infer the type based on the Autocomplete's `options`/`value`

Reply:
@bnussman-akamai , here the type can be either string or label:string, i removed this and added a check at function level